examples/nextjs/.env.local.example

```diff
@@ -0,0 +1,4 @@
+CONTENTFUL_SPACE_ID=
+CONTENTFUL_ACCESS_TOKEN=
+CONTENTFUL_PREVIEW_ACCESS_TOKEN=
+CONTENTFUL_PREVIEW_SECRET=
```

Comment:
Maybe let's add some comment explaining what each of these are, like in the templates .env.example files